spec/lib/annotate/annotate_models_spec.rb

```diff
@@ -925,16 +925,16 @@ def mock_column(name, type, options = {})
 
         let(:options) { { additional_file_patterns: additional_file_patterns } }
 
-        it do
-          expect(subject).to eq(additional_file_patterns)
+        it 'returns additional_file_patterns in the argument "options"' do
+          is_expected.to eq(additional_file_patterns)
         end
       end
 
-      context 'without additional_file_patterns' do
+      context 'when additional_file_patterns is not specified is the options' do
```

Comment:
Some typos here, but we can clean that up in a later commit :)

Reply:
Thanks, I fixed it in #731.